state/event_table.go

```diff
@@ -125,8 +125,13 @@ func NewEventTable(db *sqlx.DB) *EventTable {
 		prev_batch TEXT,
 		membership TEXT,
 		is_state BOOLEAN NOT NULL, -- is this event part of the v2 state response?
-		event BYTEA NOT NULL
+		event BYTEA NOT NULL,
+		missing_previous BOOLEAN NOT NULL DEFAULT FALSE
 	);
+
+	COMMENT ON COLUMN syncv3_events.missing_previous IS
+    'True iff the previous timeline event is not known to the proxy.';
```

Comment:
`COMMENT ON` makes it part of the postgres schema. In psql it'll show up if you describe the table with `\d+`.  For example, from Synapse:


```
matrix=> \d+ destinations
                                                                                                             Table "matrix.destinations"
             Column              │  Type  │ Collation │ Nullable │ Default │ Storage  │ Compression │ Stats target │                                                           Description                                                           
═════════════════════════════════╪════════╪═══════════╪══════════╪═════════╪══════════╪═════════════╪══════════════╪═════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════
 destination                     │ text   │           │ not null │         │ extended │             │              │ server name of remote homeserver in question
 retry_last_ts                   │ bigint │           │          │         │ plain    │             │              │ The last time we tried and failed to reach the remote server, in ms.                                                           ↵
                                 │        │           │          │         │          │             │              │ This field is reset to `0` when we succeed in connecting again.
 retry_interval                  │ bigint │           │          │         │ plain    │             │              │ How long, in milliseconds, to wait since the last time we tried to reach the remote server before trying again.                ↵
                                 │        │           │          │         │          │             │              │ This field is reset to `0` when we succeed in connecting again.
 failure_ts                      │ bigint │           │          │         │ plain    │             │              │ The first time we tried and failed to reach the remote server, in ms.                                                          ↵
                                 │        │           │          │         │          │             │              │ This field is reset to `NULL` when we succeed in connecting again.
 last_successful_stream_ordering │ bigint │           │          │         │ plain    │             │              │ Stream ordering of the most recently successfully sent PDU to this server, sent through normal send (not e.g. backfill).       ↵
                                 │        │           │          │         │          │             │              │ In Catch-Up Mode, the original PDU persisted by us is represented here, even if we sent a later forward extremity in its stead.↵
                                 │        │           │          │         │          │             │              │ See `destination_rooms` for more information about catch-up.
```

Reply:
As much as I would like to have this in-schema, this is going to fail after an upgrade because we run NewEventTable before the migrations, and `COMMENT ON COLUMN` has no `IF NOT EXISTS` option. 

Going to drop this for now.